phpspec.yml.ci

```diff
@@ -0,0 +1,10 @@
+suites:
+  data_import_suite:
+    namespace: Ddeboer\DataImport
+    psr4_prefix: Ddeboer\DataImport
+extensions:
+  - PhpSpec\Extension\CodeCoverageExtension
+formatter.name: pretty
+code_coverage:
```

Comment:
See flysystem: https://github.com/thephpleague/flysystem

PHPSpec and Unit are next to each other. Scrutinizer is able to process multiple coverage reports.